script/Ciel/Day 5/05_01_CIEL05_3C.txt

```diff
@@ -132,12 +148,14 @@ She said 'Let's talk', but it seems she already grasped what happened during the
 [sha:1a892b09d9029c82f8f9385eeb329a0be0c57628]{
 -- Page 499, Offset 12166.
 -- ……俺は今までヴローヴの根城に行ってた。おまえの忠告を無視してな。だから、俺が悪かったのは本当だよ」
-...I went to Vlov's stronghold. I ignored your warnings. So, for that, I'm really sorry."
+I've spent the whole day up until now in Vlov's stronghold...
+ I ignored your warnings.
+ That's why I meant it when I said I went off without your permission."
```

Comment:
Shouldn't this be "That's why I meant it when I said I'm sorry."

The current sentence makes it seem like the thing he wants to clarify is that he went off without permission, rather than him feeling bad about doing that.

Reply:
Yeah, I did it this way to preserve the match with the TL'd line pair but you're right, makes sense to just adjust it.